components/admin/widgets/metadata/form/MetadataForm.js

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
```

Comment:
Also, I don't see anywhere the new structure that we talked about. We talked that we were going to use Victor's way to add new components. What I mean is that you still make calls from the component instead of using actions. I know that change forms now would be complicated but please, for new components try to use it. Ok?

Reply:
Ok, I've just done the refactor of the metadata components in this commit: bbbc8255e384ab83a472d540f94844525426a1a1 
About your second comment, ok that makes sense 👍